localstack-core/localstack/aws/handlers/service.py

```diff
@@ -184,7 +185,10 @@ def create_exception_response(self, exception: Exception, context: RequestContex
             if not self.handle_internal_failures:
                 return
 
-            if config.DEBUG:
+            if (
+                os.environ.get(constants.INCLUDE_STACK_TRACES_IN_HTTP_RESPONSE)
+                in constants.TRUE_STRINGS
+            ):
```

Comment:
question: was there a specify reason that would prevent us from adding `INCLUDE_STACK_TRACES_IN_HTTP_RESPONSE` to config like other configuration variables? It would more easily discoverable, and we could still patch it in the fixture with `monkeypatch.setattr(config, "INCLUDE_STACK_TRACES_IN_HTTP_RESPONSE", True)`

But maybe I am missing something?

Reply:
Thanks for raising this 🙌  There was no strong reason not to add it to `config.py`, the flag was introduced mainly for internal use so took this approach. But I agree that making it a proper config flag would improve discoverability also since this flag affects the LocalStack behaviour. I will go ahead and move it to `config.py` with a default of `0`, and patch it to `1` in the fixture and also make relevant changes in pro 👍